packages/overlay/src/vaadin-overlay.js

```diff
@@ -98,6 +98,13 @@ class Overlay extends OverlayMixin(DirMixin(ThemableMixin(PolylitMixin(LumoInjec
     `;
   }
 
+  /** @protected */
+  firstUpdated() {
+    super.firstUpdated();
+
+    this.popover = 'manual';
```

Comment:
Since this is set here, should we remove setting the property from individual components as well?